src/components/src/test/java/org/apache/jmeter/assertions/TestJSONPathAssertion.java

```diff
@@ -66,7 +68,7 @@ public void testSetJsonValidationBool() {
     public void testIsJsonValidationBool() {
         JSONPathAssertion instance = new JSONPathAssertion();
         boolean result = instance.isJsonValidationBool();
-        assertEquals(false, result);
+        assertFalse(result);
```

Comment:
@vlsi , a contributor also spends some time which is as valuable as ours, let's not discourage contribution and be more welcoming.
@ham1  has contributed a lot

Reply:
This was done by IntelliJ so I was confident in it being correct. Apologies for the extra review effort for not very much gain.

I would love to be able to add a message explaining why it should be false, and maybe re-write in Spock ;) but there's not much to go on in the test name or comments and I'm afraid it would take too long for me to understand the full purpose of the test to add a useful message.

This was a quick change by me i.e. a few seconds to apply the IntelliJ suggestion, and I think it aids readability by a small amount. Happy to revert if it's deemed not worth the cost.